server/base/src/main/java/org/apache/accumulo/server/security/handler/ZKAuthorizor.java

```diff
@@ -87,16 +77,7 @@ public void initializeSecurity(TCredentials itw, String rootuser)
       throws AccumuloSecurityException {
     ZooReaderWriter zoo = context.getZooReaderWriter();
 
-    // create the root user with all system privileges, no table privileges, and no record-level
-    // authorizations
-    Set<SystemPermission> rootPerms = new TreeSet<>();
-    for (SystemPermission p : SystemPermission.values())
-      rootPerms.add(p);
-    Map<TableId,Set<TablePermission>> tablePerms = new HashMap<>();
-    // Allow the root user to flush the metadata tables
-    tablePerms.put(MetadataTable.ID, Collections.singleton(TablePermission.ALTER_TABLE));
-    tablePerms.put(RootTable.ID, Collections.singleton(TablePermission.ALTER_TABLE));
-
```

Comment:
It looks like these were moved to ZKPermHandler, that's why it's not needed here.